src/node_crypto.cc

```diff
@@ -5687,6 +5708,11 @@ void Initialize(Local<Object> target,
   env->SetMethodNoSideEffect(target, "certVerifySpkac", VerifySpkac);
   env->SetMethodNoSideEffect(target, "certExportPublicKey", ExportPublicKey);
   env->SetMethodNoSideEffect(target, "certExportChallenge", ExportChallenge);
+  env->SetMethodNoSideEffect(target, "loadExtraRootCertsFile",
+                             LoadExtraRootCertsFile);
+  // Exposed for testing purposes only.
+  env->SetMethodNoSideEffect(target, "isExtraRootCertsFileLoaded",
```

Comment:
Is this exposed just for testing? Should this be have a public API?

Reply:
Yes. I couldn't find a way to test this feature so that I add this API. I'm not sure if exposing API for tests is permitted or not.

    Should this be have a public API?

Did you mean that we could expose this in a "private" way, or we should not expose it?

Reply:
👍